src/components/app_download/AppDownload.module.scss

```
@@ -54,3 +54,22 @@
   right: -30px;
   z-index: 0;
 }
+.downloadBtns {
+  display: flex;
+  flex-flow: wrap;
+  justify-content: center;
+  width: 50%;
+  z-index: 2;
+  & button {
```

Thi is a similar comment to the one on pr #58 buuut
& a is the same as just having a (when nested and with a space) and I'd probably say not having them is a bit neater.
```suggestion
  button {
```